src/views/Dashboard/Customer/Home/RequestTest/PaymentReceiptDialog.vue

```diff
@@ -43,11 +43,25 @@
       div(class="text-start")
         div(class="ml-5 text-start me-10")
           div(class="d-flex justify-space-between mb-2" )
-            b( style=" font-size: 12px;" ) Total Price
+            b( style=" font-size: 12px;" ) Total Pay
             b( style="font-size: 12px;" )
               | {{  formatPrice((selectedService.price).replace(/,/g, "")) }} 
               | {{ selectedService.currency.toUpperCase()}}
 
+          div(class="d-flex justify-space-between mb-2" )
+            div( style="font-size: 10px;" ) Estimated Transaction Weight 
+              v-tooltip.visible(bottom )
+                template(v-slot:activator="{ on, attrs }")
+                  v-icon(
+                    style="font-size: 12px;"
+                    color="primary"
+                    dark
+                    v-bind="attrs"
+                    v-on="on"
+                  ) mdi-alert-circle-outline 
+                span(style="font-size: 10px;") Total fee paid in DBIO to execute this transaction.
+            div( style="font-size: 10px;" ) {{ txWeight }} milli DBIO
```

Comment:
this supposed to be dynamic (the milli text)

Reply:
i have removed the milli and use toFixed(4) instead .